docs/general/builtins/window-functions.rst

```diff
@@ -412,7 +417,7 @@ Its return type is the type of its argument.
 
 Example::
 
-   cr> SELECT col1, FIRST_VALUE(col1) OVER(ORDER BY col1) FROM unnest(['x','y', 'y', 'z']);
+   cr> SELECT col1, FIRST_VALUE(col1) OVER(ORDER BY col1) FROM (VALUES('x'), ('y'), ('y'), ('z')) as t;
```

Comment:
Would you mind capitalizing `... as ...` ? Also, maybe breaking this and the statements bellow into two lines will improve a readability? See https://github.com/crate/crate/pull/9603/files#diff-1a248cf3330381b363740c6a5ab9133dR367

Otherwise, LGTM

Reply:
Good points, thanks! Done :)